flashrom.c

```diff
@@ -1146,6 +1146,10 @@ int probe_flash(struct registered_master *mst, int startchip, struct flashctx *f
 		if (chip->bustype == BUS_SPI && !chip_to_probe && chip->spi_cmd_set != SPI25)
 			continue;
 		msg_gdbg("Probing for %s %s, %d kB: ", chip->vendor, chip->name, chip->total_size);
+
+		if (chip_to_probe && !chip->probe)
+			force = 1;
+
```

Comment:
Why is this necessary?

Reply:
with this solution, you do not have to use the force option at the command line if the chip does not support probe